pages/test_analytics/test_suites.md.erb

```diff
@@ -24,6 +24,18 @@ Organizations typically choose their main production branch as their default, al
 
 To change your default branch, go to suite settings. You can also filter Test Analytics views by any branch by typing its name into the branch query parameter in the Test Analytics URL.
 
+## Tracking down flaky tests
+
```

Comment:
I think it would be good to add some brief context about what flaky tests are here. I can't see that anywhere else yet. Thinking:

```suggestion

Flaky tests are automated tests that produce inconsistent or unreliable results, despite being run on the same code and environment. They cause frustration, decrease confidence in testing, and waste time while you investigate whether the failure is due to a genuine bug.

```